style.css

```
@@ -0,0 +1,114 @@
+body{
+  width: 100%;
+  max-width: 2180px;
+  margin: 0 auto;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size:  18px;
+  font-weight: 400;
+  line-height: 150%;
+  text-decoration: none;
+  background-color: #274001;
+  color: #BF9B6F;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  }
+  * {
+    box-sizing: border-box;
+  }
+  .header {
+    font-size: 18px;
+    flex-direction: row;
+    align-items: flex-end;
+    background-color: #8C3604;
+    flex-grow: 1;
+    flex-wrap: wrap;
+  }
+  .main {
+    display: flex;
+    align-items: center;
+    margin-top: 10px;
+  }
+  .footer {
+    padding: 20px;
+    background-color: #8C3604;
+    color: #BF9B6F;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    height: 1%;
+  }
+  .container {
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+    justify-content: space-between;
+    margin-left: 1%;
```

в процентах лучше не задавать ибо может что то поехать не так как надо 
лучше использовать неизменные величины 
```suggestion
    margin-left: 10px;
```